tree/ntuple/v7/test/ntuple_packing.cxx

```diff
@@ -353,10 +360,147 @@ TEST(Packing, OnDiskEncoding)
                                  0x04, 0x00, 0x05, 0x00, 0x06, 0x00, 0x07, 0x00};
    EXPECT_EQ(memcmp(sealedPage.GetBuffer(), expIndex64, sizeof(expIndex64)), 0);
 
+   source->LoadSealedPage(fnGetColumnId("float32Trunc"), RClusterIndex(0, 0), sealedPage);
+   // Two tightly packed 11bit floats: 0b0'01111111'11 + 0b1'10000000'11 = 0b11111111111000000011 = 0x03fe0f
+   unsigned char expF32Trunc[] = {0x03, 0xFE, 0x0F};
+   EXPECT_EQ(memcmp(sealedPage.GetBuffer(), expF32Trunc, sizeof(expF32Trunc)), 0);
+
    auto reader = RNTupleReader::Open("ntuple", fileGuard.GetPath());
    EXPECT_EQ(EColumnType::kIndex64, reader->GetModel().GetField("str").GetColumnRepresentatives()[0][0]);
    EXPECT_EQ(2u, reader->GetNEntries());
    auto viewStr = reader->GetView<std::string>("str");
    EXPECT_EQ(std::string("abc"), viewStr(0));
    EXPECT_EQ(std::string("de"), viewStr(1));
+   auto viewFtrunc = reader->GetView<float>("float32Trunc");
+   EXPECT_EQ(-3.5f, viewFtrunc(0));
+   EXPECT_EQ(1.75f, viewFtrunc(1));
```

Comment:
```suggestion
   EXPECT_FLOAT_EQ(-3.5f, viewFtrunc(0));
   EXPECT_FLOAT_EQ(1.75f, viewFtrunc(1));
```

Reply:
In this case I specifically want EQ because I know that that number has an exact representation and it should be exactly that bit pattern.